cli/tests/pcluster/validators/test_budget_validators.py

```diff
@@ -0,0 +1,136 @@
+import re
+
+import pytest
+from assertpy import assert_that
+
+from pcluster.validators.budget_validators import BudgetFilterTagValidator
+from pcluster.validators.common import FailureLevel, ValidationResult
+
+
+class FakeBoto3:
```

Comment:
Instead of define FakeBoto3 here, you can use mocker fixture, then patch the boto3 call. There are some examples in other validators. For example, you can check some examples here: aws-parallelcluster/cli/tests/pcluster/validators/test_ec2_validators.py